packages/core/src/base/staticClient.ts

```diff
@@ -80,25 +80,70 @@ export function getHeader(
 }
 
 /**
- * Format the request being made for logging.
+ * Masks property value in a json object
+ * @param key The name of the property being masked
+ * @param value Tha value to mask
+ */
+function maskInfo(key, value): string {
+  let maskedValue = value;
+  const lowerCaseKey = key.toLowerCase();
+  if (lowerCaseKey.includes("password") && value) {
+    maskedValue = "****";
+  }
+  return maskedValue;
+}
+
+/**
+ * Log request/fetch details.
  *
  * @param resource The resource being requested
  * @param fetchOptions The options to the fetch call
  */
-export const formatFetchForInfoLog = (
+export function logFetchInfo(
```

Comment:
This should just be logFetch since this is both info and debug

Reply:
By info I understood as information/details :). I will change